noxfile.py

```diff
@@ -8,6 +11,9 @@
 nox.options.default_venv_backend = "uv|virtualenv"
 nox.options.reuse_existing_virtualenvs = True
 nox.options.sessions = "lint", "tests"
+
+project = nox.project.load_toml()
+python_versions = nox.project.python_versions(project)
```

Comment:
This uses a new API: [nox.project.python_versions](https://nox.thea.codes/en/stable/config.html#nox.project.python_versions) that reads from `classifiers` from `pyproject.toml`.